playbooks/byo/preflight/README.md

```diff
@@ -0,0 +1,58 @@
+# Preflight checks
+
+Here we provide an ansible playbook (currently just one) for detecting
+potential roadblocks prior to install or upgrade.
+
+Note that currently this is only useful for RPM-based Enterprise
+installations, based on supported Red Hat RPMs. Containerized and
+Origin installs are excluded from checks for now.
+
+## Running directly
+
+For a simple trial, with an installation of Ansible 2.0 or greater,
+run the playbook directly against your inventory file:
+
+    $ git clone https://github.com/rhcarvalho/openshift-ansible.git -b pre-flight-checks
```

Comment:
I'd be okay with it pointing to openshift/openshift-ansible already, for we are unlike to revisit these details later?